query-connector/.eslintrc.json

```diff
@@ -40,7 +40,6 @@
         "publicOnly": true
       }
     ]
-    // "@typescript-eslint/no-explicit-any": "error"
```

Comment:
this was creating a json linting error for me, not sure if we want to keep?

Reply:
ahh this is a pretty important linting rule that we probably should turn on. Turning this off removes [a pretty significant advantage of TS](https://typescript-eslint.io/rules/no-explicit-any/) by giving you a catch-all escape hatch, so I feel semi-strongly for turning this on and addressing the errors accordingly.

If doing so causes a bunch of errors, we can make a ticket to address in a followup

Reply:
I'm not sure when it got commented out. gonna add it back and see if it causes too much mayhem here. if it does, yeah, let's create a new ticket.

Reply:
added ticket: https://app.zenhub.com/workspaces/skylight-dibbs-viper-6480ba23b5a07644e0e46d23/issues/gh/cdcgov/phdi/2783